### PR TITLE
fix: seed script and ferret command

### PR DESCRIPTION
**Pull request by IllusionMan1212**
yayayayayayayayayayaya

### CODE REVIEW

**Comment by IllusionMan1212** [comment]
can I get an approval on this hoe

**Comment by canarado** [comment]
I review when I get home

**Review by Sv443 — APPROVED** [review]
looks bunger to me

File: prisma/users.ts [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+    {
+        id: "407351772575694977",
+    },
+    {
+        id: "407351772575694978",
+    },
+    {
+        id: "407351772575694979",
+    },
+];
```
Comment (Sv443) [inline prisma/users.ts:307]: why so many doe

Comment (IllusionMan1212) [reply prisma/users.ts:307]: for the 2048 leaderboard paging. could cut it to 25 to get 2 pages. idk

**Review by canarado — APPROVED** [review]

**Pull request merged**